crates/iota-crypto/src/lib.rs

```diff
@@ -154,3 +154,8 @@ impl<T: Verifier<UserSignature>> IotaVerifier for T {
         self.verify(&message, signature)
     }
 }
+
+#[cfg(feature = "bech32")]
+#[cfg_attr(doc_cfg, doc(cfg(feature = "bech32")))]
+/// Bech32 prefix for IOTA private keys
```

Comment:
swap?

Reply:
actually I see that the rest of these files are like this 😤 whatever

Reply:
that's ugly indeed

Reply:
Changed the order for the bech32 methods